pom.xml

```diff
@@ -123,7 +123,7 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>gradle</artifactId>
-            <version>2.9</version>
```

Comment:
2.9 doesn't exist in https://repo.jenkins-ci.org/artifactory/releases/org/jenkins-ci/plugins/gradle/, so project wasn't building for me locally.